clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/JGroupsSubsystemXMLReader.java

```diff
@@ -154,6 +166,10 @@ private void parseChannel(XMLExtendedStreamReader reader, PathAddress subsystemA
             }
         }
 
+        if (!stackDefined) {
```

Comment:
If "stack" is a required attribute since JGroupsSchema.VERSION_4_0, shouldn't you validate that here?